feconf.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Stores various configuration options and constants for Oppia Foundation page.
```

Comment:
Let's call this file config.py. feconf.py is a misnomer that has unfortunately persisted, we should rename it at some stage. (It stands for frontend configuration but is now largely backend actually :) )